src/server/routes/revision.js

```diff
@@ -69,6 +69,16 @@ function formatCreatorChange(change) {
 		return baseFormatter.formatTypeChange(change, 'Creator Type');
 	}
 
+	if (_.isEqual(change.path, ['beginArea']) ||
+			_.isEqual(change.path, ['beginArea', 'name'])) {
+		return baseFormatter.formatAreaChange(change, 'Begin Area');
+	}
+
+	if (_.isEqual(change.path, ['endArea'] ||
+			_.isEqual(change.path, ['beginArea']))) {
```

Comment:
Two issues here:
1. path in second line is wrong
2. parenthesis placement incorrect